router/http/src/main/scala/io/buoyant/router/http/Identifier.scala

```diff
@@ -5,6 +5,13 @@ import com.twitter.finagle.buoyant.Dst
 import com.twitter.util.Future
 import io.buoyant.router.RoutingFactory
 
+object Identifier {
+  def mk(
+    prefix: Path,
+    baseDtab: () => Dtab = () => Dtab.base
+  ): RoutingFactory.Identifier[http.Request] = Identifier(prefix, false, baseDtab)
+}
+
 case class Identifier(
```

Comment:
now that we're allowing multiple Identifier types -- this should probably become BasicIdentifier or DefaultIdentifier or something that distinguishes it from other kinds.